page/layouts/shortcodes/example.html

```diff
@@ -0,0 +1,11 @@
+{{ $_hugo_config := `{ "version": 1 }` }}
```

Comment:
Not sure what that does. It is part of some of the other shortcodes…

Reply:
Seems unnecessary. :smile: I removed it in 0d29ba97d5cd9052293772be9fbd52ef3517af0d.